mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -720,8 +721,10 @@ where
                             received_by_me: tx_amounts.received_by_me,
                             // This can be 0 since it gets remapped in `coins::my_tx_history_v2`
                             my_balance_change: BigDecimal::default(),
-                            tx_hash: tx_hash.to_string(),
-                            tx_hex: msg.into(),
+                            tx: TransactionData::Signed {
+                                tx_hash: tx_hash.to_string(),
+                                tx_hex: msg.into(),
+                            },
```

Comment:
I see that we are constructing `TransactionData::Signed` a lot of time. what about creating an helper function e.g 
```rust
fn construct_signed_tx_from_hash_hex<Hash: ToString>(hash: Hash, hex: BytesJson) -> TransactionData {}
```

just some thoughts 💭

Reply:
What about `TransactionData::signed(tx_hash, tx_hex)` ? The `construct_signed_tx_from_hash_hex` name makes it even more complicated than before in my opinion.